both/_round.js

```diff
@@ -0,0 +1,175 @@
+
+// Add functions to the Mongo object, using transform (see http://docs.meteor.com/#/full/mongo_collection)
+Round = function(doc) {
+  _.extend(this, doc);
+};
+_.extend(Round.prototype, {
+
+  format: function() {
+    return wca.formatByCode[this.formatCode];
+  },
+  properties: function() {
```

Comment:
Hmm... I've never liked our "format" vs "round" name. roundFormat vs roundProperties seems better (still not in love with it, though). We don't need to do this as part of this PR, but I would like to make the naming inside of `__wca.js` consistent with this.